stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/list/adapter/internal/MessageListItemAdapter.kt

```diff
@@ -24,13 +24,21 @@ import io.getstream.chat.android.ui.feature.messages.list.adapter.BaseMessageIte
 import io.getstream.chat.android.ui.feature.messages.list.adapter.MessageListItem
 import io.getstream.chat.android.ui.feature.messages.list.adapter.MessageListItemPayloadDiff
 import io.getstream.chat.android.ui.feature.messages.list.adapter.MessageListItemViewHolderFactory
+import io.getstream.log.taggedLogger
 
 internal class MessageListItemAdapter(
     private val viewHolderFactory: MessageListItemViewHolderFactory,
 ) : ListAdapter<MessageListItem, BaseMessageItemViewHolder<out MessageListItem>>(MessageListItemDiffCallback) {
 
+    private val logger by taggedLogger("Chat:MessageListAdapter")
+
     var isThread: Boolean = false
 
+    init {
+        logger.i { "<init> isThread: $isThread" }
```

Comment:
In which case `$isThread` can be `true` at this point? :thinking:

Reply:
Yep, does't make sense to log `isThread` here as well 👍